docs/core/install/linux-alpine.md

```diff
@@ -46,15 +46,29 @@ The following versions of .NET are no longer supported. The downloads for these
 
 .NET on Alpine Linux requires the following dependencies installed:
 
+- bash
 - icu-libs
 - krb5-libs
 - libgcc
+- libgdiplus (if you expect to use System.Drawing)
```

Comment:
How about this, as the wording is similar to the other article that mentions this. Please just verify it's correct (the .Common part)

```suggestion
- libgdiplus (if the .NET app requires the *System.Drawing.Common* assembly)
```

Reply:
I was a little vague because what most people see is the namespace (System.Drawing). The package name/assembly name is System.Drawing.Common, but if your code is ported from .NET Framework, that's not familiar to you. Also, you are likely referencing the Windows Compatibility Pack, not the System.Drawing.Common package. Also, sometimes types move between assemblies, and this should be transparent. So my suggestion is to keep it in terms of namespace somehow.